packages/addon-dev/src/rollup-public-reexports.ts

```diff
@@ -0,0 +1,29 @@
+import { readJsonSync, writeJsonSync } from 'fs-extra';
+import walkSync from 'walk-sync';
+import type { Plugin } from 'rollup';
+
+export default function publicAssets(opts: { exclude: string[] }): Plugin {
+  return {
+    name: 'public-assets-bundler',
+    generateBundle() {
+      let pkg = readJsonSync('package.json');
+      const filenames = walkSync('public', {
```

Comment:
To my knowledge, the `public` folder is not something that's configurable. Otherwise, this could be part of the options (w/ a default value)